medcat/config_transformers_ner.py

```diff
@@ -13,6 +13,8 @@ class General(MixingConfig, BaseModel):
     """How many characters are piped at once into the meta_cat class"""
     ner_aggregation_strategy: str = 'simple'
     """Agg strategy for HF pipeline for NER"""
+    chunking_overlap_window: int = 5
+    """Size of the overlap window used for chunking"""
```

Comment:
As far as I can tell, changing this value does not currently change the behaviour as and when it's changed. The eval pipe of the `TransformersNER` instance would need to be recreated. Perhaps we would want to specify that in the documentation?

Reply:
We decided that this is common in different configs so if we were to tackle that, we'd need to do it in multiple places.